pkg/util/mount/mount_helper.go

```diff
@@ -44,6 +46,28 @@ func CleanupMountPoint(mountPath string, mounter Interface, extensiveMountPointC
 	return doCleanupMountPoint(mountPath, mounter, extensiveMountPointCheck, corruptedMnt)
 }
 
+// CleanupNfsMountPoint is an unmount routine that unmounts the given path awared
+// of the possibility of hang caused by lost connection with nfs server.
+func CleanupNfsMountPoint(mountPath string, mounter Interface, extensiveMountPointCheck bool) error {
+	pathExists, pathErr := PathExistsWithTimeout(mountPath, 1*time.Second)
+	if pathErr == wait.ErrWaitTimeout {
+		klog.Warningf("Timeout checking %q, unmounting", mountPath)
+		if err := mounter.Unmount(mountPath); err != nil {
```

Comment:
why add additional unmount here? will unmount hang here?

Reply:
First umount will hang here, and a succeeding umount could break that